man/rgblink.5

```diff
@@ -56,51 +50,109 @@ and
 .Ql \et .
 Other backslash escape sequences in
 .Xr rgbasm 5
-are only relevant to assembly code and do not apply in section names.
+are only relevant to assembly code and do not apply in linker scripts.
+.Ss Directives
+.Bl -tag -width Ds
+.It Including other files
+.Ql Ic INCLUDE Ar path
+acts as if the contents of the file at
+.Ar path
+were copy-pasted in place of the
+.Ic INCLUDE
+directive.
+.Ar path
+must be a string.
+.It Specifying the active bank
+.Tg region
+The active bank can be set by specifying its type (memory region) and number.
+The possible types are:
+.Ic ROM0 , ROMX , VRAM , SRAM , WRAM0 , WRAMX , OAM ,
+and
+.Ic HRAM .
+However, if the type only contains a single bank, then the number can be omitted.
+The applicable types are:
```

Comment:
And this is just overall confusing. If you're going to list the types for which bank numbers are required, I'd split this entire section into a new paragraph; however, that's probably not necessary at all. (If you want to keep the current structure, just restate the purpose of the list: "The bank types for which the bank number can be omitted are...".)

Reply:
How about: "For types that only contain a single bank, the number can be omitted. These types are:"?

Reply:
Or: "The bank number can be omitted from the types which only contain a single bank which are:"